docs/source/support/release-notes.rst

```diff
@@ -57,6 +57,11 @@ Sandbox
 
 - Added recovery around failing ledger entry persistence queries using Postgres. See `#1505 <https://github.com/digital-asset/daml/pull/1505>`__.
 
+Ledger
+~~~~~~
+
+- Introduced a new API for party management.
```

Comment:
This sounds like a significant addition, is there any documentation to go with it?

Reply:
Good point. @bitonic Do we want to add the new API endpoints to the public docs, presumably at https://docs.daml.com/app-dev/services.html?

Reply:
Yes, however not in this PR. I'd keep it on the low for now until things have stabilized.